core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/AuditDataBinderImpl.java

```diff
@@ -51,9 +52,11 @@ public AuditEntryTO getAuditTO(final AuditEntry auditEntry) {
         }
 
         if (auditEntry.getInput() != null) {
-            auditTO.getInputs().addAll(Arrays.stream(auditEntry.getInput()).
-                    map(input -> ToStringBuilder.reflectionToString(input, ToStringStyle.JSON_STYLE)).
-                    collect(Collectors.toList()));
+            auditTO.getInputs().addAll(Arrays.stream(auditEntry.getInput())
+                .map(input -> input == null
```

Comment:
shouldn't be better to add `filter()` before to skip null input?

Reply:
Let me confirm, is `null` a valid input in the `getInput()` array? an audit entry might in fact receive a null value as an input parameter?

Reply:
Generally speaking, yes; not for user / group / any object CRUD, of course.